recipes/r-sandpaper/meta.yaml

```diff
@@ -0,0 +1,124 @@
+{% set version = '0.16.11' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-sandpaper
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/carpentries/sandpaper/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 2036381b8421f2e47afd8cfc6be56fa0aab70877da7bd773476a431a3f7b2517
+  patches:
+    # c.f. https://github.com/carpentries/sandpaper/pull/651
+    - add-lesson-example-to-config-template-source.patch
```

Comment:
This patch from https://github.com/carpentries/sandpaper/pull/651 is needed only to get the tests to fully pass.

c.f. the discussion in PR https://github.com/conda-forge/staged-recipes/pull/29866 (specifically https://github.com/conda-forge/staged-recipes/pull/29866#issuecomment-2844839784).